app-modules/task/src/Filament/Resources/TaskResource.php

```diff
@@ -46,15 +46,15 @@ class TaskResource extends Resource
 {
     protected static ?string $model = Task::class;
 
-    protected static ?string $navigationGroup = 'Service Management';
+    protected static ?string $navigationGroup = 'Project Management';
 
     protected static ?int $navigationSort = 90;
```

Comment:
Do you need to update the sort number to reflect its new position?

Reply:
Sorry @kandarp-canyon you did push changes but you didn't change the sort number